packages/patternfly-4/react-core/src/components/DataList/DataListAction.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import styles from '@patternfly/patternfly-next/components/DataList/styles.css';
+import { EllipsisVIcon } from '@patternfly/react-icons';
+import { Button } from '../Button';
+
+const DataListAction = ({ className, ...props }) => (
+  <div className={css(styles.dataListAction, className)} {...props}>
+    <Button variant="plain" aria-label="Actions">
```

Comment:
No problem but just wanted to hear your opinion.
It seems like `Dropdown` is more suitable here than `Button`?
What do you think?

Reply:
If we use the `Dropdown` in the example, I assume then it would show a menu on click? If that's the case, then I think a `Dropdown` would be better.

Also, does this implementation allow the consumer to use a single action button if that's what their design calls for?

Reply:
@jgiardino - yeah that's what I thought. I will change the implementation to use `Dropdown`.
Also, I will work on the case when there is a single action.

Thanks for your help!